cmd/network/network_templates.go

```diff
@@ -0,0 +1,139 @@
+package network
+
+import (
+	"strconv"
+	"text/template"
+)
+
+const wpaTemplate = `
```

Comment:
Is it possible that we put followings into a tpl file？
https://github.com/rancher/os/blob/master/images/01-base/Dockerfile

Then we can read  this file as template, execute it and write it back.